accounts/abi/bind/backends/simulated.go

```diff
@@ -225,7 +225,11 @@ func (b *SimulatedBackend) callContract(ctx context.Context, call ethereum.CallM
 	from.SetBalance(common.MaxBig)
 	// Execute the call.
 	msg := callmsg{call}
-	vmenv := core.NewEnv(statedb, chainConfig, b.blockchain, msg, block.Header(), vm.Config{})
+
+	evmContext := core.ToEVMContext(msg, block.Header(), b.blockchain)
```

Comment:
@fjl also mentioned it and I didn't want to change it then. But now that you're the second person mentioning this I'll change this to `NewEVMContext`.